### PR TITLE
add topic "motorola-68000"

### PR DESCRIPTION
**Pull request by Wladefant**
### Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: <https://github.com/github/explore/blob/main/CONTRIBUTING.md>.
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288, size <= 75 kB)
- [ ] Content (and my changes are in `index.md`)

> Motorola 68000 is a microprocessor that has multiple assemblers, compilers, and operating systems. It is used in many personal computers, workstations, and game consoles, including the Apple Macintosh, Atari ST, Commodore Amiga, and the Nintendo Entertainment System.

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [ ] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: <https://github.com/github/explore/tree/main/docs>

> This topic is not curated right now and has no explanations

### Something that does not neatly fit into the binary options above

- [ ] My suggested edits are not about an existing topic or collection, or at least not a single one
- [ ] My suggested edits are not about curating a new topic or collection, or at least not a single one
- [ ] My suggested edits conform to the Style Guide and API docs: https://github.com/github/explore/tree/main/docs

> see index.md

---

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

**Review by Wladefant — COMMENTED** [review]
deleted specification of logo.
deleted alias that is the same as topic name.

File: topics/motorola-68000/index.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+aliases: m68k, 68k, m68000, 68000, 68000-series, 68000-family, 68000-processor, mc68000, motorola-680x0
+created_by: Motorola
+display_name: Motorola 68000
+related: assembly, cpu, microprocessor, processor, 68k-assembly, cpm-68000, assembly-68000, cpm-68k, motorola-6809, motorola-68020, motorola-68030, motorola-68040, motorola-68060, m68k-atari-mint, 6800, m68k-assembly, m68k-disassembler, m68k-assembly, m68k-elf-gcc, assembly-m68k, m68k-emulation, motorola-6805
```
Comment (kenyonj) [inline topics/motorola-68000/index.md:5]: duplicated
```suggestion
related: assembly, cpu, microprocessor, processor, 68k-assembly, cpm-68000, assembly-68000, cpm-68k, motorola-6809, motorola-68020, motorola-68030, motorola-68040, motorola-68060, m68k-atari-mint, 6800, m68k-disassembler, m68k-assembly, m68k-elf-gcc, assembly-m68k, m68k-emulation, motorola-6805
```

File: topics/motorola-68000/index.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
```
Comment (kenyonj) [inline topics/motorola-68000/index.md:6]: no need for this blank line
```suggestion
```

**Review by kenyonj — CHANGES_REQUESTED** [review]
Thanks for your contribution! There are some changes that need to be made for CI.

**Comment by kenyonj** [comment]
Looks like the related list has to be trimmed down now:

```
should have no more than 10 related topics
```

File: topics/motorola-68000/index.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+aliases: m68k, 68k, m68000, 68000, 68000-series, 68000-family, 68000-processor, mc68000, motorola-680x0
+created_by: Motorola
+display_name: Motorola 68000
+related: 68k-assembly, assembly-68000, m68k-atari-mint, m68k-assembly, m68k-disassembler, m68k-assembly, m68k-elf-gcc, assembly-m68k, m68k-emulation, motorola-6805
+
```
Comment (kenyonj) [inline topics/motorola-68000/index.md:6]: `m68k-assembly` is still duplicated and there is still a blank line, removing both here:
```suggestion
related: 68k-assembly, assembly-68000, m68k-atari-mint, m68k-assembly, m68k-disassembler, m68k-elf-gcc, assembly-m68k, m68k-emulation, motorola-6805
```

**Review by kenyonj — CHANGES_REQUESTED** [review]

**Comment by Wladefant** [comment]
Ready?

**Review by kenyonj — APPROVED** [review]
Looks great. Thanks for your contribution!

**Pull request merged**